src/cpu/cpu-exec.c

```diff
@@ -47,7 +47,7 @@ rtlreg_t tmp_reg[4];
 
 #ifdef CONFIG_DEBUG
 static inline void debug_hook(vaddr_t pc, const char *asmbuf) {
-  log_write("%s\n", asmbuf);
+  Logti("%s\n", asmbuf);
```

Comment:
What does Logti mean?

Reply:
I improve NEMU log function. So "log_write"  can write information into file or printf information into screen(default). In order to control whether the informaiton is printed in "debug_hook", I replace log_write with Logti. Beside, we can use Log() to output information to screen or file when we use "-l file_name" in NEMU command line.